pkg/specialservices/service_syncer.go

```diff
@@ -53,14 +53,18 @@ func SyncKubernetesService(
 		return err
 	}
 
+	// detect if cluster ips changed
+	clusterIPsChanged := vObj.Spec.ClusterIP != pObj.Spec.ClusterIP || !equality.Semantic.DeepEqual(vObj.Spec.ClusterIPs, pObj.Spec.ClusterIPs)
```

Comment:
Makes sense :) pushed!

Reply:
Also, it currently seems to update. I'm not sure why...

Reply:
checking

Reply:
Seems like it's because something resets the ports before we update it here when the syncer starts. I think think the IPFamilyPolicy only really needs to sync if the ClusterIPs change so I think this is fine

Reply:
That something is probably the controller-manager that also owns this service, but does not update it